stock_picking_invoicing/README.rst

```diff
@@ -97,6 +97,7 @@ Contributors
 * Florent THOMAS <florent.thomas@mind-and-go.com>
 * François Honoré <francois.honore@acsone.eu>
 * Magno Costa <magno.costa@akretion.com.br>
+* Luis Felipe Mileo <mileo@kmee.com.br>
```

Comment:
Hi @mileo thanks for improvement,  you should to put this information on CONTRIBUTORS.rst file and then use the oca-gen-addon-readme command to generated it automatically https://github.com/oca/maintainer-tools Besides it everything looks good for me, it'll be better if you can included tests, but this thing is not problem to approve for me.

Reply:
Done

Reply:
I'm on another task now, I think I can only do that tomorrow.

Could you do the favor and add the tests? I think it's just duplicating one of the tests you already know and setting the max_picking2invoice_lines parameter to a value that divides the picking.

Thanks